features/events/syslog/src/main/java/org/opennms/netmgt/syslogd/SyslogParser.java

```diff
@@ -141,9 +144,12 @@ protected static Date parseDate(final String dateString) {
             // Date pattern has been crearted and checked inside if loop instead of 
             // parsing date inside the exception class.
             if (dateString.matches(datePattern)) {
-                final DateFormat df = new SimpleDateFormat("yyyy-MM-dd", Locale.ROOT);
-                df.setTimeZone(TimeZone.getTimeZone("UTC"));
-                return df.parse(dateString);
+            	
+              LocalDate date = LocalDate.parse(dateString);
+              DateTimeFormatter df = DateTimeFormatter.ofPattern("yyyy-MM-dd",Locale.ROOT);
+              date.format(df);
```

Comment:
Is th comment regarding usage of dateformatter.I have used date formatter in Oder to set it as Locale.Root to support local date,I would remove it if not required .